app/ts/library/identifyTokens.ts

```diff
@@ -58,12 +60,12 @@ export async function itentifyAddress(address: string, id: bigint, provider: Pro
 		},
 		{
 			target: address,
-			callData: nftInterface.encodeFunctionData('supportsInterface', ['0xd9b67a26']) // Is ERC721Metadata
-		},
-		{
-			target: address,
-			callData: nftInterface.encodeFunctionData('supportsInterface', ['0x0e89341c']) // Is ERC721Metadata
+			callData: nftInterface.encodeFunctionData('supportsInterface', ['0xd9b67a26']) // Is ERC1155
 		},
+		// {
```

Comment:
old code?

Reply:
Well this check *SHOULD* be here, to follow the EIP1155Metadata spec, but OpenSea's 1155 contract doesn't implement the whole metadata spec.
Do I leave this here as reference, follow the spec and show no metadata for OpenSea shared storefront items or not make the check as the `uri(uint256 id)` will return either return a string or fail the call either way.
CC @MicahZoltu

Reply:
Solved on call